### PR TITLE
Checking tests

### PR DESCRIPTION
**Pull request by n-lina**
this branch is not actually relating to checking tests

### CODE REVIEW

File: frontend/app/screens/goal-detail-screen/goal-detail-screen.tsx [hunk 1]
```diff
@@ -219,13 +231,13 @@ export const GoalDetailScreen = observer(function GoalDetailScreen() {
...
 
   for (const goal of STgoals) {
-    for (let i = 0; i < goal.mon.length; i++) monday.push([goal.mon[i], goal.title, goal.id]);
-    for (let i = 0; i < goal.tue.length; i++) tuesday.push([goal.tue[i], goal.title, goal.id]);
-    for (let i = 0; i < goal.wed.length; i++) wednesday.push([goal.wed[i], goal.title, goal.id]);
-    for (let i = 0; i < goal.thu.length; i++) thursday.push([goal.thu[i], goal.title, goal.id]);
-    for (let i = 0; i < goal.fri.length; i++) friday.push([goal.fri[i], goal.title, goal.id]);
-    for (let i = 0; i < goal.sat.length; i++) saturday.push([goal.sat[i], goal.title, goal.id]);
-    for (let i = 0; i < goal.sun.length; i++) sunday.push([goal.sun[i], goal.title, goal.id]);
+    if (goal.mon) for (let i = 0; i < goal.mon.length; i++) monday.push([goal.mon[i], goal.title, goal.id]);
```
Comment (jeremywiens) [inline frontend/app/screens/goal-detail-screen/goal-detail-screen.tsx:234]: Do we need to create another PR for this change? Is this if statement needed?

File: frontend/app/models/goals-store/goals-store.ts [hunk 2]
```diff
@@ -3,175 +3,175 @@ import { Goal, GoalModel } from "../goal/goal";
...
-  STgoals: cast([{ id: "", title: "Cook meals at home", mon: [100], tue: [30], wed: [4] },
-    { id: "", title: "Exercise 15 minutes", mon: [100], tue: [30], wed: [4] },
-    { id: "", title: "Go to sleep early", mon: [100], tue: [30], wed: [4] },
-    { id: "", title: "Wake up early", mon: [100], tue: [30], wed: [4] },
-    { id: "", title: "Drink more water", mon: [100], tue: [30], wed: [4] },
-    { id: "", title: "Eat more fruit and vegetables", mon: [100], tue: [30], wed: [4] },
-    { id: "", title: "Buy whole grain and high protein products", mon: [100], tue: [30], wed: [4] },
+  STgoals: cast([{ id: "", title: "Cook meals at home", mon: [1080], wed: [1080], fri: [1080] },
+    { id: "", title: "Exercise 15 minutes", mon: [600], tue: [600], wed: [600] ,thu: [600], fri: [600]},
+    { id: "", title: "Go to sleep early", mon: [1320], tue: [1320], wed: [1320], thu:[1320], fri:[1320], sat: [1320], sun: [1320]},
```
Comment (jeremywiens) [inline frontend/app/models/goals-store/goals-store.ts:11]: Do you think we should add these changes to main in another PR?

File: frontend/app/screens/goal-detail-screen/goal-detail-screen.tsx (hunk 1 above)
Comment (n-lina) [reply frontend/app/screens/goal-detail-screen/goal-detail-screen.tsx:238]: i got an error when going from common-goals to the detail screen, it's there for safety reasons. do you think we should just edit the common goals to evade this error or keep it for code safety?

File: frontend/android/settings.gradle [hunk 3]
```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'DailyDash'
+include ':react-native-share'
```
Comment (jeremywiens) [inline frontend/android/settings.gradle:2]: Can you remove these changes?

File: frontend/app/screens/awards-screen/awards-screen.tsx [hunk 4]
```diff
@@ -82,26 +81,17 @@ const styles = StyleSheet.create({
...
 
-  // let myAwards = []
-
-  // async function fetchAwards(awardArr: any[]) {
-  //   setRefreshing(true);
-  //   awardArr = await userStore.getAwards()
-  //   console.log(awardArr)
-  //   setRefreshing(false);
-  // };
+  const onBackPress = () => navigation.navigate("profile");
```
Comment (jeremywiens) [inline frontend/app/screens/awards-screen/awards-screen.tsx:93]: Can you remove this line?

File: frontend/app/screens/awards-screen/awards-screen.tsx [hunk 5]
```diff
@@ -82,26 +81,17 @@ const styles = StyleSheet.create({
   }
 });
 
+const BACK_BUTTON: ViewStyle = {
```
Comment (jeremywiens) [inline frontend/app/screens/awards-screen/awards-screen.tsx:84]: This is also a duplicate

File: frontend/app/screens/awards-screen/awards-screen.tsx [hunk 6]
```diff
@@ -82,26 +81,17 @@ const styles = StyleSheet.create({
...
+  backgroundColor: palette.white,
+};
+
 export const AwardsScreen = observer(function AwardsScreen() {
   // Pull in one of our MST stores
   const { userStore } = useStores();
+  const navigation = useNavigation();
```
Comment (jeremywiens) [inline frontend/app/screens/awards-screen/awards-screen.tsx:91]: This is also a duplicate
